docs/README.md

```diff
@@ -0,0 +1 @@
+## Extended man pages for CLI commands
```

Comment:
These aren't for public use right? Should we move these to tools/docgen?

Reply:
hmm I'm considering moving docgen/main.go to `docs` actually, but no strong opinion here. Main motivation is to follow this [folder structure](https://github.com/golang-standards/project-layout/tree/master/docs). Also see this openshift example https://github.com/openshift/origin/blob/master/docs/cli.md.

Reply:
Sgtm as long as it's clear that it's not user-facing

Reply:
We can allow users to contribute to these markdown docs. This will be dumped to our docs site eventually. See this screenshot https://user-images.githubusercontent.com/1639722/213419223-3fc58030-19ec-4396-849e-6d68f2a3cad6.png